File: SERVIDOR SDR_SCR/lib/domain/dto/response/get_attendance_control_list_response.py
from lib.domain.entities.attendace_control_lists import AttendanceControlLists

class GetAttendanceControlListResponse:
    def __init__(self, attendanceControlLists: AttendanceControlLists):
        self.__attendanceControlLists = attendanceControlLists

    def getAttendanceControlLists(self) -> AttendanceControlLists:
        return self.__attendanceControlLists

File: SERVIDOR CENTRAL/lib/domain/dto/response/get_attendance_control_list_response.go
package response

import "gitlab.com/jorgechavezrnd/servidorseminariogo2/lib/domain/entities"

// GetAttendanceControlListResponse ...
type GetAttendanceControlListResponse struct {
	StudentsPresentSlice                   []entities.Student `json:"students_present_list"`
	StudentsMissingSlice                   []entities.Student `json:"students_missing_list"`
	NumberOfDoubtfulStudents               string             `json:"number_of_doubtful_students"`
	PictureWithFacesDetectedAndNamesBase64 string             `json:"picture_with_faces_detected_and_names_base_64"`
}
